src/pages/CitiesPage/CitiesSection.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import CitiesList from '../../components/common/CitiesList/CitiesList';
+import { cities } from '../сitiesData';
+
+const CitiesSection = () => {
+  const initialLimit = 20;
+  const limitStep = 10;
+  const [limit, setLimit] = useState(initialLimit);
+
+  const increaseLimit = () => {
```

Comment:
Now `increaseLimit` is the function passed in the component and it should be wrapped into `useCallback` to avoid passing different references to `CitiesList`